GuestBook/Andy.txt

```diff
@@ -0,0 +1 @@
+Andrew (Andy) Hazelton 7/24/23
```

Comment:
Let's do a date format change like Month Day, Year (e.g. July 24, 23).

Reply:
I've pushed an update to the date.